bin/compile

```diff
@@ -1,5 +0,0 @@
-#!/bin/bash
-
-# bin/compile is currently required, even though it's deprecated. This is likely a bug.
-
-exit
```

Comment:
I don't think we can just delete this one, unfortunately ... I'm actually surprised we haven't hit any issues with this deletion.

[this commit](https://github.com/cyberark/cloudfoundry-conjur-buildpack/commit/44b592e95af5a069ab02b3a8d0798de803ed0789) explains why we left this useless file

from a slack convo around that time, I noted:
> when I try to cf create-buildpack it is failing on the error:
>   Failed to create a local temporary zip file for the buildpack
>   FAILED
>   Couldn't write zip file: Zip archive does not contain a buildpack
> it looks like this could be due to an expectation that I will include bin/compile: https://github.com/cloudfoundry/cli/blob/1f695102e83c7bb5e09fb6cd5602abf745433782/cf/api/buildpack_bits.go#L194
> but that is listed as deprecated here: https://docs.cloudfoundry.org/buildpacks/understand-buildpacks.html#buildpack-scripts 

it appears that the CLI still uses `bin/compile` to determine the buildpack directory :/

Reply:
Is it possible that's no longer the case? `cf create-buildpack` currently works without issue. Or is it just safer to keep the file in, just in case?

Reply:
I'm not sure. I did ask in the partners slack to see what they would say - I'll flag the original thread there for you too, in case it helps to read up